core/Cargo.toml

```diff
@@ -233,11 +234,15 @@ parity-scale-codec = { version = "3.6.9", default-features = false }
 subxt-signer = { version = "0.39.0", default-features = false }
 
 # Celestia
-celestia-types = "0.6.1"
+celestia-types = "0.12"
 bech32 = "0.11.0"
 ripemd = "0.1.3"
 tonic = { version = "0.11.0", default-features = false }
 pbjson-types = "0.6.0"
+# eq-sdk = "0.1.8"
+# eq-sdk = {git="https://github.com/celestiaorg/eq-service.git", rev="33702d5a649dbc76b5fba975dcd869aff3962e21"}
+eq-sdk = {path = "../../eq-service/sdk"}
```

Comment:
Guess it's obvious, but we will need to publish and link up to the latest `eq-sdk` before merging.

Reply:
fixed